misc/multihash/src/hashes.rs

```diff
@@ -60,6 +63,7 @@ impl Hash {
     /// Get the hash length in bytes.
     pub fn size(&self) -> u8 {
         match self {
+            Hash::Identity => 42,
```

Comment:
Also note that this is entirely redundant. A go implementation uses this: 
https://github.com/multiformats/go-multihash/blob/64ffd94071a5bc58130ea88fd8404408aef993f2/multihash.go#L141